Nos/Views/HomeFeedView.swift

```diff
@@ -107,9 +108,32 @@ struct HomeFeedView: View {
                         .padding()
                 }
             })
-            .navigationBarItems(leading: SideMenuButton())
+            .toolbar {
+                ToolbarItem(placement: .navigationBarLeading) {
+                    SideMenuButton()
+                }
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Button {
+                        router.sideMenuPath.append(SideMenu.Destination.relays)
+                        router.toggleSideMenu()
+                    } label: {
+                        HStack(spacing: 3) {
+                            Image("relay-left")
+                                .colorMultiply(numberOfConnectedRelays > 0 ? .white : .red)
+                            Text("\(numberOfConnectedRelays)")
```

Comment:
Did you try binding this Text directly to `relayService.numberOfConnectedRelays`? RelayService is an ObservableObject so I think we should be able to do that.

Reply:
👌